subprojects/ide/src/main/java/org/gradle/plugins/ide/eclipse/internal/EclipseProjectMetadata.java

```diff
@@ -20,31 +20,31 @@
 import org.gradle.api.tasks.TaskProvider;
 import org.gradle.internal.Describables;
 import org.gradle.internal.DisplayName;
-import org.gradle.plugins.ide.eclipse.model.EclipseProject;
+import org.gradle.plugins.ide.eclipse.model.EclipseModel;
 import org.gradle.plugins.ide.internal.IdeProjectMetadata;
 
 import java.io.File;
 import java.util.Collections;
 import java.util.Set;
 
 public class EclipseProjectMetadata implements IdeProjectMetadata {
-    private final EclipseProject eclipseProject;
+    private final EclipseModel eclipseProject;
     private final File projectDir;
     private final TaskProvider<? extends Task> generatorTask;
 
-    public EclipseProjectMetadata(EclipseProject eclipseProject, File projectDir, TaskProvider<? extends Task> generatorTask) {
+    public EclipseProjectMetadata(EclipseModel eclipseProject, File projectDir, TaskProvider<? extends Task> generatorTask) {
         this.eclipseProject = eclipseProject;
         this.projectDir = projectDir;
         this.generatorTask = generatorTask;
     }
 
     @Override
     public DisplayName getDisplayName() {
-        return Describables.withTypeAndName("Eclipse project", eclipseProject.getName());
+        return Describables.withTypeAndName("Eclipse project", eclipseProject.getProject().getName());
     }
 
     public String getName() {
-        return eclipseProject.getName();
+        return eclipseProject.getProject().getName();
```

Comment:
Why this change?

Reply:
The type of the field has changed (to give access to `EclipseClasspath`). I've just renamed the field to avoid confusion.